HACKING.md

```diff
@@ -184,3 +186,39 @@ tag if it was created.
 * Flutter is currently a fork to add support for line numbers. We need to
   figure out a cleaner way of doing this so we can merge it into the
   upstream codebase.
+
+## Glossary
+
+<dl>
+  <dt>reStructuredText</dt>
+  <dd>A markup language identified with the Python ecosystem.</dd>
+
+  <dt>Abstract Syntax Tree (AST)</dt>
+  <dd>A tree of nodes which reflect the syntactic structure of an unparsed textual document.</dd>
+
+  <dt>Postprocessor</dt>
+  <dd>The component of snooty which performs global link analysis and other forms of processing that spans multiple files.</dd>
+
+  <dt>docutils</dt>
+  <dd>The canonical parsing library for reStructuredText. We use our own vendored fork called `tinydocutils`, modified to be statically typed and less reliant on method dispatch through string manipulation.</dd>
```

Comment:
The backticks for inline code don't seem to be parsed here. Would we need to use `<code>`?

Reply:
Ah, drat. Markdoooown!!! *shakes fist* Good catch thanks!

Reply:
Nice. I believe `tt` is considered deprecated and `code` is preferred, but this may be more of a nitpick at this point

Reply:
Never mind, I'm just le silly!